Getting-Started/Setup/Upgrading/upgrading-to-v8.md

```diff
@@ -0,0 +1,80 @@
+---
+versionFrom: 8.1.0
+---
+# Migrating content to Umbraco 8
+
+Umbraco 8 contains a lot of breaking changes and a lot of code has been cleaned up compared to Umbraco 7. Due to this, it will not be possible to do a simple upgrade from Umbraco 7 to Umbraco 8. Instead you will need to **migrate your content** from your Umbraco 7 site into your Umbraco 8 site and then recreate the rest in the new version.
+
+A content migration tool has been implemented in Umbraco 8.1.0, to help you with the transition.
+
+In this guide you can read more about the tool, it's limitations and how to use it in practice.
+
+## Limitations
+
+### Versions supported
+
+The content migration tool is a database migration, which is made for the database schema of Umbraco 7.14+. This means that in order to do the migration the first step is to ensure your Umbraco 7 site is running at least Umbraco 7.14.
+
+### Database types supported
+
+As Umbraco 8 doesn't support MySQL databases the migration will not work when moving from an Umbraco 7 site using MySQL to Umbraco 8 on SQL server for example.
+The database types that are supported are SQL Server and SQL CE.
+
+### Third party property editors
+
+The migration will transform the data stored in third party editors as well, however it will simply be stored as it was in Umbraco 7. If the structure has changed for Umbraco 8 or the property editor doesn't exist you will still be able to find the data in the database, but you would not see it in the backoffice.
+
+## What will happen?
+
+When the migrations are running, Umbraco will go through your entire Umbraco 7 database and update it to the format required for Umbraco 8. The schema will be remodelled and transformed into the correct format and your existing compatible data will also be transformed to fit with how Umbraco 8 is storing data.
+
+Please be aware that these migrations will be running directly on your database. They are transforming schema and data - not transferring. Therefore always ensure that you have a backup before attempting to do this. In case something goes wrong, you will be able to rollback and try again.
```

Comment:
Suggest a dry run, and timing how long it takes, for when you do this for real, so you'll know how long to advise in terms of content freeze